queries/ecma/highlights.scm

```diff
@@ -22,7 +22,7 @@
 ; Special identifiers
 ;--------------------
 ((identifier) @type
-  (#lua-match? @type "^[A-Z]"))
+  (#lua-match? @type "^[A-Z].+?[a-z]"))
```

Comment:
For a lua pattern with lazy quantifiers I believe this would need to use `.-`? Although it means zero or more, not one or more, so maybe the pattern would need a slight adjustment